app/react/portainer/settings/SettingsView/KubeSettingsPanel/KubeConfigSection.tsx

```diff
@@ -0,0 +1,45 @@
+import { useField } from 'formik';
+
+import { FormControl } from '@@/form-components/FormControl';
+import { FormSection } from '@@/form-components/FormSection';
+import { PortainerSelect } from '@@/form-components/PortainerSelect';
+
+const options = [
+  {
+    label: '1 day',
+    value: '24h',
+  },
+  {
+    label: '7 days',
+    value: `${24 * 7}h`,
+  },
+  {
+    label: '30 days',
+    value: `${24 * 30}h`,
+  },
+  {
+    label: '1 year',
+    value: `${24 * 30 * 12}h`,
+  },
+  {
+    label: 'No expiry',
+    value: '0',
+  },
+] as const;
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

Reply:
this gives options a specific value that we can later use literally. so the first object in the array will always be `{label:'1 day', value:'24h'}` instead of `{label:string, value:string}`.

not so important here but usually when I have a constant value that is not dependent on another value I use `as const`

![image](https://github.com/portainer/portainer/assets/1381655/8d52295d-9702-45a5-8e16-c8c79a39c33a)